.ado/templates/e2e-test-job.yml

```diff
@@ -85,7 +104,7 @@ jobs:
       - task: CopyFiles@2
         displayName: Copy tree dump output files
         inputs:
-          sourceFolder: $(LocalAppData)\Packages\ReactUWPTestApp_cezq6h4ygq1hw\LocalState
+          sourceFolder: $(UserProfile)\Documents\ReactUWPTestApp_cezq6h4ygq1hw!App
```

Comment:
just to double check that the PFN matches the PFN from the PFX file (here and elsewhere)

Reply:
@asklar  This is what I see in the Package.appxmanifest file:
ReactUWPTestApp_cezq6h4ygq1hw
Is this what you meant?

Reply:
yes - the cez... part is created by hashing the app name and the publisher and probably other stuff too. So the Publisher name used in the PFX must match so that the PFN of the app ends up being ReactUWPTestApp_cezq6h4ygq1hw
If it doesn't match then the app will be installed elsewhere and you'll need to update the path in the YML as well as SetupLocalDumps.cmd